README.rst

```diff
@@ -40,7 +40,7 @@ Usage
 First, enable middleware in your settings.py::
 
     DOWNLOADER_MIDDLEWARES = {
-       'scrapy_po.InjectPageObjectsMiddleware': 543,
+       'scrapy_po.InjectionMiddleware': 543,
```

Comment:
I changed the name because I think the Middleware is doing more than just injecting PageObjects... it is injecting many things. But probably is also influenced because I have not clear the "Page Object" concept now that it can also deal with components.

Reply:
I like the new name 👍